builtin/k8s/plugin.proto

```diff
@@ -28,3 +28,8 @@ message Resource {
     string name = 1;
   }
 }
+
+// Task represents the state of the TaskLaunch implementation.
+message Task {
```

Comment:
The Docker plugin defines this as `TaskInfo`, should we be consistent or is that not very relevant?

Reply:
Doesn't hurt to match it.